services/src/main/java/org/fao/geonet/api/UserProfileUtil.java

```diff
@@ -0,0 +1,35 @@
+package org.fao.geonet.api;
```

Comment:
you need to include general license in the top of the file.

Reply:
I see a `UserUtil`, apologies as didn't see that initially. Maybe these methods can be added in that class instead of a new one.

Reply:
@josegar74 

No problem, its a small refactoring.